libs/platform/src/lib/components/list/action-list-item/action-list-item.component.ts

```diff
@@ -0,0 +1,46 @@
+import {
+    Component, ChangeDetectorRef, ElementRef,
+    ChangeDetectionStrategy, forwardRef, Input
+} from '@angular/core';
+import { ENTER, SPACE } from '@angular/cdk/keycodes';
+import { Router } from '@angular/router';
+
+import { ListConfig } from '../list.config';
+import { BaseListItem } from '../base-list-item';
+
+@Component({
+    selector: 'fdp-action-list-item',
+    templateUrl: './action-list-item.component.html',
+    providers: [
+        { provide: BaseListItem, useExisting: forwardRef(() => ActionListItemComponent) }
+    ],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class ActionListItemComponent extends BaseListItem {
+
+    @Input()
+    title: string;
+
```

Comment:
Does this component emit an event on button click? If it doesn't we should add that feature.